scripts/mods/mod_reforged/dynamic_spawns/parties/barbarian_parties.nut

```diff
@@ -9,7 +11,8 @@ local parties = [
 		UnitBlockDefs = [
 			{ BaseID = "UnitBlock.RF.BarbarianFrontline", RatioMin = 0.60, RatioMax = 1.00, DeterminesFigure = true },	// Vanilla: doesn't care about size
 			{ BaseID = "UnitBlock.RF.BarbarianSupport", RatioMin = 0.00, RatioMax = 0.07, ReqPartySize = 10 },			// Vanilla: Start spawning in armies of 15+; At 24+ a second drummer spawns
-			{ BaseID = "UnitBlock.RF.BarbarianDog", RatioMin = 0.00, RatioMax = 0.15, ReqPartySize = 5 },		// Vanilla: Start spawning in armies of 6+
+			{ BaseID = "UnitBlock.RF.BarbarianDog", RatioMin = 0.00, RatioMax = 1.00, },		// Vanilla: Start spawning in armies of 6+
```

Comment:
I don't understand raising the max for dogs. What do you want to achieve with that?
That will spawn potentially more dogs, when your comment says you don't actually want many dogs in those parties